packages/SystemsThinkingWorkbench-Core.package/STWNode.class/instance/addArrowFrom.to..st

```diff
@@ -0,0 +1,11 @@
+interaction
+addArrowFrom: startingNode to: endNode 
```

Comment:
Is there any case where this does not operate with `endNode = self`?
Maybe this argument is redundant.